Makefile

```diff
@@ -122,22 +122,16 @@ diff_cover: test
 
 ## Localization targets
 
-extract_translations: ## extract strings to be translated, outputting .mo files
+extract_translations: ## extract strings to be translated, outputting .po files
```

Comment:
I believe this pair of comments is incorrect. `makemessages` generates `.po` files, and `compilemessages` runs over the `.po` files and generates `.mo` files.